src/Confluent.Kafka/Impl/LibRdKafka.cs

```diff
@@ -58,7 +58,9 @@ internal enum AdminOp
             DeleteTopics = 2,
             CreatePartitions=  3,
             AlterConfigs = 4,
-            DescribeConfigs = 5
+            DescribeConfigs = 5,
+            DeleteRecords = 6,
+            DeleteGroups = 7
```

Comment:
can you put the other ones supported by librdkafka here too? looks like I forgot to do it with DeleteRecords

```
        ....
        RD_KAFKA_ADMIN_OP_DELETEGROUPS,     /**< DeleteGroups */
        /** DeleteConsumerGroupOffsets */
        RD_KAFKA_ADMIN_OP_DELETECONSUMERGROUPOFFSETS,
        RD_KAFKA_ADMIN_OP_CREATEACLS,   /**< CreateAcls */
        RD_KAFKA_ADMIN_OP_DESCRIBEACLS, /**< DescribeAcls */
        RD_KAFKA_ADMIN_OP_DELETEACLS,   /**< DeleteAcls */
```